src/commands.c

```diff
@@ -383,6 +383,19 @@ static int32_t execute_flash( dfu_device_t *device,
     if( mem_type == mem_user ) {
         result = atmel_user( device, &bout );
     } else {
+		if ( 1 == args->com_flash_data.erase_first ) {
```

Comment:
Hmmm, something is not quite right with the indentation here.

Eventually I'd like to have some automatic code formatting implemented.

Reply:
ups, tabs vs spaces - Will fix

Reply:
fixed now